src/averageAges.js

```diff
@@ -37,7 +41,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const peoples = people.filter(
+    peop => withChildren !== undefined
+      ? peop['sex'] === 'f' && people.some(
+        human => human['mother'] === peop['name']
+      )
+      : peop['sex'] === 'f'
```

Comment:
```suggestion
    peop => 
      withChildren
        ? peop.sex === 'f' && people.some(
          human => human.mother === peop.name
        )
        : peop.sex === 'f'
```
Не потрібно використовувати квадратні дужки для доступу до ключа, якщо ти точно знаєш його назву